packages/webpack-plugin/src/plugin.ts

```diff
@@ -295,40 +295,38 @@ export class StylableWebpackPlugin {
         const options =
             loadStylableConfig(compiler.context, (config) => {
                 return isWebpackConfigProcessor(config)
-                    ? config.webpackPlugin(defaults, compiler)
+                    ? config.webpackPlugin(defaults, compiler, getTopLevelInputFilesystem(compiler))
                     : undefined;
             })?.config || defaults;
 
         this.options = options;
     }
-    private createStcBuilder(compiler: Compiler) {
-        if (!this.options.stcConfig) {
-            return;
-        }
-
+    private getStylableConfig(compiler: Compiler) {
         const configuration = resolveStcConfig(
             compiler.context,
-            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined
+            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined,
+            getTopLevelInputFilesystem(compiler)
```

Comment:
why we need the fs here?

Reply:
I think I know why....because we evaluate the default config inside...I'm not sure we want this...

Reply:
The reasoning seems right, we can re-think this if we want.